core-bundle/src/DependencyInjection/Configuration.php

```diff
@@ -112,6 +112,15 @@ static function (array $options): array {
                         ->end()
                     ->end()
                 ->end()
+                ->arrayNode('pagination')
```

Comment:
Do we really need a configuration for that? I can't think of a use case where I want to adjust this to `8` on the entire system. Maybe I want to adjust it for a special element or so but I have never ever adjusted this value. In the backend you could adjust it super easy by adjusting the template now. So for KISS purposes I would just remove it. I know we could previously adjust that value and it might be considered a BC break but personally, I would favor having a simpler config (= no config for things that don't need to be configured) over considering this very, very rare case of people who adjusted this. I think most people don't even know that because there's no visual configuration field in `tl_settings.php` 😊

Reply:
I agree too. If someone _really_ needs to change that, use a compiler pass to adjust the factory.